### ISSUE TITLE
Icons not showing

### ISSUE THREAD

**Issue by Danita** [MikeFielden/ng-compass-boilerplate#1]
First of all, thank you for porting ng-boilerplate to compass! I've installed it following the instructions on the Readme file, and after `grunt watch` everything runs fine, but the example page doesn't show any icons.

I'm a newbie at this. The `/dist` directory is very criptic to me and I don't know where to look to fix the issue. Any ideas?

**Comment by Danita** [comment]
It seemed that not every bootstrap component was imported on `main.scss`. I've added `@import '../../vendor/sass-bootstrap/lib/_sprites.scss';` and at least the icon classes were present on output, but the images referenced couldn't be found nowhere in the directory structure. I copied them from `/vendor/sass-bootstrap/img` to `/src/assets/img` and it seemed to fix the issue, but still there are icons missing from the front page ("Complete build system", etc). 

Now I'm realising that the original ng-boilerplate includes font-awesome which it seems ng-compass-boilerplate doesn't. Is that so?

**Comment by MikeFielden** [comment]
@Danita Yeah I pulled out Font-Awesome with the intention of adding it back and never got around to it :) Sorry about that... Pull requests are welcome however... Thanks for the issue!

**Comment by Danita** [comment]
Thanks Mike :)
